recipes/rangesnext/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "cci.20210426":
+    url: "https://github.com/cor3ntin/rangesnext/archive/10fd1cffe7a2a9688a31f91ec8754668c4cdd9a6.zip"
+    sha256: efb222ba840215ac6b239ff3b2446be5a2ac76f874ea547390df33ab1c673109 
```

```suggestion
    url: "https://github.com/cor3ntin/rangesnext/archive/10fd1cffe7a2a9688a31f91ec8754668c4cdd9a6.tar.gz"
    sha256: 8b4ba9129346818d1ad2ab1fe00ccb85c5c1b35fef91868aece97aa079e0dec4 
```

Prefer tar.gz, it's smaller than .zip and usually linux distros don't have unzip installed by default.